web-common/src/features/dashboards/filters/measure-filters/measure-filter-utils.ts

```diff
@@ -39,21 +40,24 @@ export function splitWhereFilter(whereFilter: V1Expression | undefined) {
   const dimensionThresholdFilters: DimensionThresholdFilter[] = [];
   whereFilter?.cond?.exprs?.filter((e) => {
     const subqueryExpr = e.cond?.exprs?.[1];
-    if (subqueryExpr?.subquery) {
-      const filters = isJoinerExpression(subqueryExpr.subquery.having)
-        ? (subqueryExpr.subquery.having?.cond?.exprs
-            ?.map(mapExprToMeasureFilter)
-            .filter(Boolean) as MeasureFilterEntry[])
-        : [mapExprToMeasureFilter(subqueryExpr.subquery.having)];
-
-      dimensionThresholdFilters.push({
-        name: subqueryExpr.subquery.dimension ?? "",
-        filters: filters.filter(Boolean) as MeasureFilterEntry[],
-      });
+    if (!subqueryExpr?.subquery?.having) {
+      dimensionFilters.cond?.exprs?.push(e);
       return;
     }
 
-    dimensionFilters.cond?.exprs?.push(e);
+    const isValidMeasureFilter =
+      !isJoinerExpression(subqueryExpr.subquery.having) ||
```

Comment:
What's this condition mean? I'm not clear what a "joiner" expression is. Looking only at this line, I'm not sure it's right that a valid measure filter is equal to any expression that is not a "joiner"? 

What about including something like this:
```typescript
whereFilter?.cond?.exprs?.forEach((expr) => {
    if (isMeasureFilter(expr)) {
      dimensionThresholdFilters.push(createDimensionThresholdFilter(expr));
    } else {
      dimensionFilters.cond?.exprs?.push(expr);
    }
```

Reply:
I have renamed "joiner" expression to clarify what it is. A measure filter supported in UI could be something like `mes > 10` this is definitely supported right now and it not a "joiner".